next-app/src/routes/password/change/+page.svelte

```diff
@@ -16,6 +17,15 @@
 	$: if (form?.failed) {
 		$error = Error(form.failed)
 	}
+
+	$: new_password_confirm && debounce(() => verify(new_password_confirm, new_password), 400)
+
+	$: submittable = new_password && new_password_confirm && new_password_confirm === new_password
+	$: disabled = ! submittable
+
+	function verify(password_confirm: string, password: string) {
+		password_confirm !== password ? $error = Error('Passwords are not the same') : dismiss()
```

Comment:
Instead of an error bubble, I think it would ultimately be nicer to have the error directly under the relevant field. This is a special case, where it could ultimately be either field that's "wrong", but I think it would still be fine to just put it under the confirmation field.

Reply:
agreed, when it comes to forms, not utilizing the app-wide error solution would be better for the user.